### ISSUE TITLE
copy button is not displayed for longer code block

### ISSUE THREAD

**Issue by zclab** [jbms/sphinx-immaterial#55]
I found that there is no copy button for longer code block which is 40 lines or more.

**Expected behaviour**

<img width="762" alt="code2" src="https://user-images.githubusercontent.com/13864359/161177429-a103d676-84e5-46c0-8312-5d6e5a4d0bb4.png">

**Actual behaviour**

<img width="762" alt="code1" src="https://user-images.githubusercontent.com/13864359/161177477-8c15cdc1-0477-408d-ae9d-5a57033b44aa.png">

I use: Python 3.8.9, sphinx 4.4.0 and sphinx_material 0.4.0

**Comment by 2bndy5** [comment]
It looks like the doc-string's indentation is different by a level. I doubt that would fix this though. What was the last version number with the button had worked for you?

**Comment by zclab** [comment]
It worked on sphinx_material 0.3.1

**Comment by 2bndy5** [comment]
I noticed that this happens with the auto-docs' generated source code pages

![image](https://user-images.githubusercontent.com/14963867/162855595-e53bc882-7138-41bd-a59e-9b2d26ff9f95.png)
vs
![image](https://user-images.githubusercontent.com/14963867/162855692-6d3e190a-2b89-48ae-b8ae-44cd91899693.png)

Still looking into this...

**Comment by 2bndy5** [comment]
Well, this isn't a problem in mkdocs-material, but it definitely is something we merged in upstream v8.2.5 - I suspect it is something to do with the changes made to mounting code blocks for large pages (a speed up to satisfy https://github.com/squidfunk/mkdocs-material/issues/3643).

TL;DR - Still not exactly sure what we merged from upstream that is causing this issue.

**Comment by jbms** [comment]
This is actually an upstream bug that was fixed here:
https://github.com/squidfunk/mkdocs-material/commit/88cfda42882e3ac23c1d3e0d2ad292a2105a7b75

We just need to merge in the updates.

**Comment by jbms** [comment]
Note: The best way to check if something is an upstream bug is to reproduce using the version of mkdocs-material listed in `MKDOCS_MATERIAL_MERGE_BASE`.

**Comment by 2bndy5** [comment]
yeah, that was my bad. Even more motivation to resolve the merge problems with recently v8.2.8+.
